homeassistant/util/unit_conversion.py

```diff
@@ -269,6 +277,7 @@ class SpeedConverter(BaseUnitConverterWithUnitConversion):
         SPEED_MILES_PER_HOUR,
         SPEED_MILLIMETERS_PER_DAY,
     }
+    _PINT_UNIT_MAP = {SPEED_KNOTS: "knot"}
```

Comment:
`kn` is not recognised as a valid unit in `pint`, so need manual mapping.

Reply:
https://pint.readthedocs.io/en/stable/defining.html

I think we can easily extend pints units.

Reply:
Yes, it works for knots:
`_UNIT_REGISTRY.define("kn = knot")`

However fluid ounces does not, it keeps converting to `<UnitsContainer({'femtoliter': 1, 'ounce': 1})>`